src/display/editor/editor.js

```diff
@@ -1336,6 +1336,17 @@ class AnnotationEditor {
     return editor;
   }
 
+  /**
+   * Check if an existing annotation associated with this editor has been
+   * modified.
+   * @returns {boolean}
+   */
+  get hasBeenModified() {
+    return (
+      this.annotationElementId && (this.deleted || this.serialize() !== null)
```

Comment:
As mentioned recently in another review comment, please take care to ensure that the return value is *consistent* in getters!
```suggestion
      !!this.annotationElementId && (this.deleted || this.serialize() !== null)
```